Makefile

```diff
@@ -293,8 +293,8 @@ sample-conf-check:
 	for flag in $$(GO_FLAGS_COMPLETION=1 go run -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd -- | grep -v help | cut -c3-); do if ! grep -q $$flag sample-lnd.conf; then echo "Command line flag --$$flag not added to sample-lnd.conf"; exit 1; fi; done
 
 mobile-rpc:
-	@$(call print, "Creating mobile RPC from protos (prefix=$(prefix)).")
-	cd ./lnrpc; COMPILE_MOBILE=1 SUBSERVER_PREFIX=$(prefix) ./gen_protos_docker.sh
+	@$(call print, "Creating mobile RPC from protos.")
```

Comment:
We should update the docs that his is no longer needed. 

Does this change the API for the existing apps using the non-prefix version?

Reply:
This does change existing apps, yes. At least if they use the non-prefixed version. But given that there are collisions if you don't use the prefix, maybe nobody was able to use the non-prefixed code for a while and there is nobody affected?

Reply:
Thinking of the most used LND mobile apps right now:

- Breez: forked, has its own build system
- Blixt: already using prefix

Any others come to mind we should check out?

Reply:
We could post a question in the #mobile channel on Slack. Maybe we'll get some more insight there?

Reply:
Good idea

Reply:
I'm going to merge this in the meantime. We can always roll this back during the RC phase if anyone has issues with the prefix.